FirebaseRemoteConfig/Sources/RCNConfigExperiment.m

```diff
@@ -131,6 +151,9 @@ - (void)updateExperimentsWithHandler:(void (^)(NSError *_Nullable))handler {
                            lastStartTime:lastStartTime
                                 payloads:_experimentPayloads
```

Comment:
so alternatively we could call `updateActiveExperimentsInDB` before this block, then pass `_activeExperimentPayloads` here? 

do you think that would be better? right now `_activeExperimentPayloads` isn't used anywhere iiuc (though i guess we will use it once we add diff-ing for realtime

Reply:
We could do that but I think it's fine the way it is. We plan on adding diff-ing for real-time right after this anyways and real-time will be the main user of `_activeExperimentPayloads`

Reply:
ok sg